extensions/wikia/AdEngine/js/config/desktop.js

```diff
@@ -62,6 +64,10 @@ define('ext.wikia.adEngine.config.desktop', [
 			return [];
 		}
 
+		return [
+			adProviderNetzAthleten
+		];
+
 		// Force provider
 		if (context.forcedProvider && !!forcedProviders[context.forcedProvider]) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this code after the "return" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnreachableCode)